backend/src/app/rpc/commands/teams.clj

```diff
@@ -815,6 +851,58 @@
 
         itoken))))
 
+(defn- add-user-to-team
+  [conn profile team email role]
+
+  (let [team-id (:id team)
+        member (db/get* conn :profile
+                        {:email (str/lower email)}
+                        {:columns [:id :email]})
+        params (merge
+                {:team-id team-id
+                 :profile-id (:id member)}
+                (role->params role))]
+
+      ;; Do not allow blocked users to join teams.
+    (when (:is-blocked member)
+      (ex/raise :type :restriction
+                :code :profile-blocked))
+
+    (quotes/check-quote! conn
+                         {::quotes/id ::quotes/profiles-per-team
+                          ::quotes/profile-id (:id member)
+                          ::quotes/team-id team-id})
+
+    ;; Insert the member to the team
+    (db/insert! conn :team-profile-rel params {::db/on-conflict-do-nothing? true})
+
+    ;; Delete any request
+    (db/delete! conn :team-request
+                {:team-id team-id :requester-id (:id member)})
+
+    ;; Delete any invitation
+    (db/delete! conn :team-invitation
+                {:team-id team-id :email-to (:email member)})
+
+    (eml/send! {::eml/conn conn
+                ::eml/factory eml/join-team
+                :public-uri (cf/get :public-uri)
+                :to email
+                :invited-by (:fullname profile)
+                :team (:name team)
+                :team-id (:id team)})))
+
+
+(def sql:valid-requests-email
+  "select p.email from team_request tr, profile p
+     where tr.team_id = ?
+       and tr.auto_join_until > now()
+       and tr.requester_id = p.id")
```

Comment:
1. use proper and explicit join
2. format sql in this way:
![image](https://github.com/user-attachments/assets/4575ea7e-b4fe-4597-b268-65a4e8a5c0d3)
all new sql should be formatted using this style
3. Use proper sql aliases with `AS`

Reply:
fixed